src/applications/disability-benefits/view-payments/tests/components/Payments/helpers.js

```diff
@@ -0,0 +1,163 @@
+export const mockData = {
```

Comment:
Can this file be removed? Doesn't appear to be used anywhere now.

Reply:
Yep, removed this.